client/TwitterWall.js

```diff
@@ -0,0 +1,20 @@
+(function() {
```

Comment:
This is good, but could be split up the same way as the other module and named explicitly (`angular.module("TwitterWallAdminApp", [])`) - that said it works just fine as is so feel free to merge and I'll do the minor stuff as I'm working on it.